scripts/derive_field_constants.py

```
@@ -0,0 +1,243 @@
+# Pasta moduli + generator, read from arkworks ark-pallas / ark-vesta.
+# pallas base Fq = vesta scalar Fr = P ; pallas scalar Fr = vesta base Fq = Q.
+# --------------------------------------------------------------------------
+
+P = 28948022309329048855892746252171976963363056481941560715954676764349967630337
+Q = 28948022309329048855892746252171976963363056481941647379679742748393362948097
+PASTA_GENERATOR = 5
+
+
+def emit_limbs(name, x):
+  limbs = to_limbs(x)
+  body = ", ".join(f"UINT64_C({l})" for l in limbs)
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)